chaos/net-rpc-client.go

```diff
@@ -75,10 +76,7 @@ func (rpcClient *ReconnectRPCClient) dialRPCClient() (*rpc.Client, error) {
 }
 
 // Call makes a RPC call to the remote endpoint using the default codec, namely encoding/gob.
-func (rpcClient *ReconnectRPCClient) Call(serviceMethod string, args interface {
```

Comment:
> Let me add a token based Login/Logout method in chaos test for sake of understanding.

Write it outside of chaos in a separate directory as an example which talks about authenticated lock client.